fw/http.c

```diff
@@ -6756,10 +6759,14 @@ tfw_http_resp_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 	 * @hmsib is not attached to the connection yet.
 	 */
 	r = tfw_http_resp_cache(hmresp);
-	if (unlikely(r != T_OK))
+	if (unlikely(r != T_OK)) {
+		if (hmsib)
```

Comment:
In case of error where we don't drop the connection we must not freed `hmsib`

Reply:
This function always return error after which we drop the connection